tests/plugins/clipboard_figure.test.ts

```diff
@@ -13,71 +14,77 @@ import {
   updateChart,
 } from "../test_helpers/commands_helpers";
 import { getCellContent } from "../test_helpers/getters_helpers";
+import { getFigureDefinition, getFigureIds, nextTick } from "../test_helpers/helpers";
 import { BarChartDefinition } from "./../../src/types/chart/bar_chart";
 
-describe("Clipboard for figures", () => {
+describe.each(["chart", "image"])("Clipboard for image & chart figures", (type: string) => {
```

Comment:
jest will replace `%s` by "chart" and "image" at runtime
```suggestion
describe.each(["chart", "image"])("Clipboard for %s figures", (type: string) => {
```